web/styles/settings.css

```diff
@@ -1786,7 +1788,25 @@ label.preferences-radio-choice-label {
 
     #settings_page {
         .settings-header.mobile {
-            display: block;
+            display: flex;
+            justify-content: space-between;
+            height: 45px;
+
+            .fa-chevron-left {
+                position: static;
+                transform: translate(-50px, 0);
+                opacity: 0;
+                color: hsl(0deg 0% 67%);
+                padding: 2px 10px;
+                cursor: pointer;
+                transition: none 0.3s ease;
+                transition-property: opacity, transform;
+            }
```

Comment:
I moved this out of `subscriptions.css` and removed the relative positioning stuff.